hudi-utilities/src/test/resources/delta-streamer-config/source-jdbc.avsc

```diff
@@ -26,34 +26,42 @@
   },
   {
 	"name": "TIMESTAMP",
-	"type": ["double", "null"]
+	"type": ["null", "double"],
```

Comment:
why change this. are tests failing?

Reply:
Yes. This schema is used as Expected schema in one of the tests. Changed the schema to the evolved schema.